#!/usr/bin/env python
PACKAGE = "perception"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("start_loc_x", int_t, 0,                 "Start location (x-axis) of the ROI area relative to recorded image size.", 0, 0, 1280)
gen.add("start_loc_y", int_t, 0,                 "Start location (y-axis) of the ROI area relative to recorded image size.", 219, 0, 1024)
gen.add("end_loc_x", int_t, 0,                   "End location (x-axis) of the ROI area relative to recorded image size.", 752, 0, 1280)
gen.add("end_loc_y", int_t, 0,                   "End location (y-axis) of the ROI area relative to recorded image size.", 479, 0, 1024)

exit(gen.generate(PACKAGE, "perception", "PreprocessingRoi"))
